addons/netfox/rollback/rollback-synchronizer.gd

```diff
@@ -84,13 +83,24 @@ func process_authority():
 			_record_input_props.push_back(pe)
 			_auth_input_props.push_back(pe)
 
+func process_time_flags():
+	_latest_state = NetworkTime.tick - 1
+	_earliest_input = NetworkTime.tick
+	if (_started_at_tick == -1):
+		_started_at_tick = NetworkTime.tick
+
 func _ready():
 	process_settings()
+
+	if (await_first_frame):
+		await get_tree().process_frame
+	process_authority()
```

Comment:
After `await`, it is basically guaranteed the input authority is set, so `process_authority` can be safely invoked